packages/ui5-application-writer/package.json

```diff
@@ -27,11 +27,11 @@
         "json-merger": "1.1.6"
     },
     "devDependencies": {
-        "@types/ejs": "3.0.6",
+        "@types/ejs": "3.1.0",
         "@types/mem-fs": "1.1.2",
         "@types/fs-extra": "9.0.13",
         "@types/mem-fs-editor": "7.0.0",
-        "typescript": "4.2.4",
+        "typescript": "4.4.4",
```

Comment:
@devinea: Is it not sufficient to have this at the root?

Reply:
Should be, don't recall why they were added in the packages. Removed in https://github.com/SAP/open-ux-tools/pull/163/commits/24d69e71333a00927b1bff619e79e6e1a7875531